src/gobject-introspection.mk

```diff
@@ -0,0 +1,27 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := gobject-introspection
+$(PKG)_WEBSITE  := https://gi.readthedocs.io
+$(PKG)_DESCR    := GObject introspection is a middleware layer between C libraries (using GObject) and language bindings.
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 1.70.0
+$(PKG)_CHECKSUM := 902b4906e3102d17aa2fcb6dad1c19971c70f2a82a159ddc4a94df73a3cafc4a
+$(PKG)_SUBDIR   := gobject-introspection-$($(PKG)_VERSION)
+$(PKG)_FILE     := gobject-introspection-$($(PKG)_VERSION).tar.xz
+$(PKG)_URL      := https://download.gnome.org/sources/gobject-introspection/$(call SHORT_PKG_VERSION,$(PKG))/$($(PKG)_FILE)
+$(PKG)_DEPS     := cc meson-wrapper
+$(PKG)_TARGETS  := $(BUILD) $(MXE_TARGETS)
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'https://gitlab.gnome.org/GNOME/gobject-introspection/tags' | \
+    $(SED) -n "s,.*<a [^>]\+>v\?\([0-9]\+\.[0-9.]\+\)<.*,\1,p" | \
+    $(SORT) -Vr | \
+    head -1
+endef
+
+define $(PKG)_BUILD
```

Comment:
Something I have to point out: you must use `$(PKG)_BUILD_$(BUILD)` for native compilation. `$(PKG)_BUILD` is invoked once per target architecture, while `$(PKG)_BUILD_$(BUILD)` is built for native only once.

You also must use

```
$(PKG)_DEPS    := $(BUILD)~$(PKG)
$(PKG)_DEPS_$(BUILD) := cc meson-wrapper
```

So appropriate architecture is used for deps too.

We don't need gobject-introspection, it seems, but I wanted to note mistakes so you won't repeat them in further commits.

Reply:
Oh, and by the way - prefer `git pull --rebase` when conflicting with upstream. It reduces noise in commits.

Reply:
I agree, I will use `git pull --rebase` from now onwards as well. 

Ah indeed. I should use `$(PKG)_BUILD_$(BUILD)`. I think I was confused with the ATK code.

Reply:
ATK does not build for local machine, only for target, that's why it uses different code.

Reply:
BTW on that note. When I execute make with `MXE_TARGETS='x86_64-w64-mingw32.static.posix'` parameter, I also noticed that glib (incl. a lot of other packages) are build for the `x86_64-pc-linux-gnu` target as well. Is that really necessary? I mean I just want to cross-compile GTK/Glib.

Reply:
It is needed for some packages, but I'm not sure why GLib does that. Will look into this later.